pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -1445,6 +1476,7 @@ private ByteBuf processMessageChunk(ByteBuf compressedPayload, MessageMetadata m
                 }
                 chunkedMsgCtx.recycle();
                 chunkedMessagesMap.remove(msgMetadata.getUuid());
+                increaseAvailablePermits(cnx);
```

Comment:
Should it be increaseAvailablePermits(cnx, chunkedMsgCtx.chunkedMessageIds.size()).